### ISSUE TITLE
[Issue]: About 1 out of 40 generated images results in a black image.

### ISSUE THREAD

**Issue by Ancanus** [vladmandic/automatic#1468]
### Issue Description

This seems trivial, but it makes EbSynth completely unusable as it's guaranteed at least one frame will be ruined by the black image corruption and ruin the exported movie.

Please help me figure out what information I can provide to debug this. I am able to breakpoint into code.

### Version Platform Description

```
17:42:52-520690 INFO     Starting SD.Next
17:42:52-524691 INFO     Python 3.10.9 on Windows
17:42:52-562687 INFO     Version: 5808b7a6 Sat Jun 17 08:13:19 2023 -0400
17:42:52-899686 INFO     Latest published version: b340e3f40c36eda26bf62fcbcdfec7219a731cea 2023-06-18T20:30:59Z
17:42:52-902685 INFO     nVidia CUDA toolkit detected
17:42:54-493686 INFO     Torch 2.0.0+cu118
17:42:54-512685 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
17:42:54-514685 INFO     Torch detected GPU: NVIDIA GeForce RTX 4070 Ti VRAM 12281 Arch (8, 9) Cores 60
```

### Acknowledgements

- [X] I have read the above and searched for existing issues

**Comment by vladmandic** [comment]
run in fp32 or no-half (either will cost a lot of performance), change cross attention method (it may or may not help), select better model (there are sooo many borderline models), etc...

sorry, but this is due to math rounding problems in model and not something i can do much about.
and its one of the most common problems since first days of stable diffusion.

i can keep the issue open if someone wants to chime in, but thats about it.

**Comment by Ancanus** [comment]
`BF16` seems to have fixed it for me!

**Comment by Aptronymist** [comment]
> ### Issue Description
> This seems trivial, but it makes EbSynth completely unusable as it's guaranteed at least one frame will be ruined by the black image corruption and ruin the exported movie.
> 
> Please help me figure out what information I can provide to debug this. I am able to breakpoint into code.
> 
> ### Version Platform Description
> ```
> 17:42:52-520690 INFO     Starting SD.Next
> 17:42:52-524691 INFO     Python 3.10.9 on Windows
> 17:42:52-562687 INFO     Version: 5808b7a6 Sat Jun 17 08:13:19 2023 -0400
> 17:42:52-899686 INFO     Latest published version: b340e3f40c36eda26bf62fcbcdfec7219a731cea 2023-06-18T20:30:59Z
> 17:42:52-902685 INFO     nVidia CUDA toolkit detected
> 17:42:54-493686 INFO     Torch 2.0.0+cu118
> 17:42:54-512685 INFO     Torch backend: nVidia CUDA 11.8 cuDNN 8700
> 17:42:54-514685 INFO     Torch detected GPU: NVIDIA GeForce RTX 4070 Ti VRAM 12281 Arch (8, 9) Cores 60

Have you tried the Enable Upcast Sampling? Upcast cross attention layer? We're not really seeing many of your settings with just the log there.

You may wish to upgrade your torch for one, so you're up to 2.0.1. Couldn't hurt.
Also I'm pretty sure you could just manually yank that image out if you get a nan and just dupe the previous or next frame to fill in the spot, or is it ruining the entire stream because it's interpolating from the NAN image?
I haven't used EbSynth much, just a tiny bit months ago, but if you used the same seed and ran it again, couldn't you get the same output, which would likely fail in a different place? Can't you restart the generation process from where it failed?
